src/Modal/Modal.js

```diff
@@ -83,35 +86,61 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
 
   const handleContentClick = event => event.stopPropagation();
 
-  const handleBackdropClick = () => {
-    if (!props.closeOnBackdropClick) return;
-
-    handleClose();
-  };
-
-  useKeyPress('Escape', () => {
-    if (!isOpen || !props.closeOnEscape) return;
-    handleClose();
-  });
-
   useEffect(() => {
     if (open !== isOpen) {
       setOpen(open);
     }
   }, [open]);
 
+
+  useEffect(() => {
+    if (blur) {
+      const targets = document.querySelectorAll(blurTargets);
```

Comment:
Let's drop the blur - this is overly complex for a small luxury with less than great performance and support